scss/6-components/forms/checkboxes.scss

```diff
@@ -0,0 +1,20 @@
+.cads-checkbox-group {
```

Comment:
Might be tempted to use `cads-form-group--checkbox` to signal it's a BEM modifier, which is what this looks like. Although tbh this reads OK to me as well.

Reply:
yup good shout